### PR TITLE
Morgue tray messages

### PR DESCRIPTION
**Pull request by 9600bauds**
Alt title: "i try to do something small and end up doing code housekeeping"

Adds this message for ghosted players when their corpse gets put into a morgue tray
dreamseeker_2016-07-30_01-49-52
Fixes live mobs not being able to exit morgue trays

Unfucks morgue code
Removes snowflake, removing mystery "canclone" var in the process

### CODE REVIEW

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -37,67 +37,88 @@
...
 
+/obj/structure/morgue/examine(mob/user)
+	..()
+	switch(icon_state)
+		if("morgue2")
+			to_chat(user, "<span class='info'>\The [src]'s light display indicates there is a catatonic corpse inside.</span>")
+		if("morgue3")
+			to_chat(user, "<span class='info'>\The [src]'s light display indicates there are items inside.</span>")
+		if("morgue4")
+			to_chat(user, "<span class='info'>\The [src]'s light display indicates there is an active corpse inside.</span>")
```
Comment (9600bauds) [inline code/game/objects/structures/morgue.dm]: If anyone has any ideas for a more immulshions-friendly message I'm listening

Comment (Exxion) [inline code/game/objects/structures/morgue.dm]: Something about trace brain activity, maybe?

Comment (9600bauds) [inline code/game/objects/structures/morgue.dm]: Maybe just "potential cloning candidate"?

Comment (ComicIronic) [inline code/game/objects/structures/morgue.dm]: clone-compatible

**Comment by SonixApache** [comment]
As someone that checks the morgue lights often, this is great.

File: code/game/dna/dna_modifier.dm [hunk 2]
```diff
@@ -293,6 +291,14 @@
 
 	return 1
 
+/obj/machinery/dna_scannernew/on_login(var/mob/M)
+	if(M.mind && !M.client && locate(/obj/machinery/computer/cloning) in range(src, 1)) //!M.client = mob has ghosted out of their body
+		var/mob/dead/observer/ghost = get_ghost_from_mind(M.mind)
+		if(ghost && ghost.client)
+			ghost << 'sound/effects/adminhelp.ogg'
+			to_chat(ghost, "<span class='interface'><b><font size = 3>Your corpse has been placed into a cloning scanner. Return to your body if you want to be resurrected/cloned!</b> \
```
Comment (PJB3005) [inline code/game/dna/dna_modifier.dm]: `<font size = 3>`

Deprecated since HTML 4. REMOVE.

File: code/game/objects/structures/morgue.dm [hunk 3]
```diff
@@ -37,65 +37,87 @@
...
+			if(prob(5))
+				for(var/atom/movable/A in src)
+					A.forceMove(src.loc)
+					A.ex_act(severity)
 				qdel(src)
 				return
-	return
 
-/obj/structure/morgue/alter_health()
+/obj/structure/morgue/alter_health() //???????????????
```
Comment (PJB3005) [inline code/game/objects/structures/morgue.dm:73]: Ask the oldcoders?

File: code/game/dna/dna_modifier.dm (hunk 2 above)
Comment (9600bauds) [inline code/game/dna/dna_modifier.dm]: uuurgh it's the same for all cloning messages I'll fix it some other time

File: code/game/objects/structures/morgue.dm [hunk 4]
```diff
@@ -37,65 +37,87 @@
...
+				A.forceMove(src.loc)
+				A.ex_act(severity)
 			qdel(src)
 			return
 		if(2.0)
-			if (prob(50))
-				for(var/atom/movable/A as mob|obj in src)
-					A.loc = src.loc
-					ex_act(severity)
+			if(prob(50))
```
Comment (ComicIronic) [inline code/game/objects/structures/morgue.dm:59]: What did I tell you about non-standard formatting changes?

Comment (Exxion) [inline code/game/objects/structures/morgue.dm:59]: Fuck you and fuck spaces before parentheses

Comment (ComicIronic) [inline code/game/objects/structures/morgue.dm:59]: I will close this PR young man, so help me God

Comment (9600bauds) [inline code/game/objects/structures/morgue.dm:59]: I touched the entire fuckin rest of the proc, if that's not good enough then so help me

File: code/game/dna/dna_modifier.dm (hunk 2 above)
Comment (ghost) [inline code/game/dna/dna_modifier.dm]: no more half measures, bauds.

File: code/game/objects/structures/morgue.dm (hunk 4 above)
Comment (PJB3005) [inline code/game/objects/structures/morgue.dm:59]: SPACES BETWEEN KEYWORDS AND PARENTHESES.

Comment (9600bauds) [inline code/game/objects/structures/morgue.dm:59]: You are wrong

Comment (clusterfack) [inline code/game/objects/structures/morgue.dm]: Jesus christ people

**Comment by ComicIronic** [comment]
Don't get an attitude with me in those commit names, young man. I'll turn this repo around.

**Comment by 9600bauds** [comment]
Does that involve reverting x64?

**Comment by ComicIronic** [comment]
It involves reverting every commit since r4407.

**Pull request merged**